mm2src/lp_bot/simple_market_maker.rs

```diff
@@ -0,0 +1,266 @@
+use crate::mm2::lp_bot::{SimpleCoinMarketMakerCfg, SimpleMakerBotRegistry, TickerInfosRegistry, TradingBotContext};
+use crate::mm2::lp_ordermatch::{retrieve_my_maker_orders, MakerOrder};
+use common::{executor::{spawn, Timer},
+             log::{error, info},
+             mm_ctx::MmArc,
+             mm_error::MmError,
+             slurp_url, HttpStatusCode};
+use derive_more::Display;
+use http::{HeaderMap, StatusCode};
+use serde_json::Value as Json;
+use std::collections::HashSet;
+use std::{str::Utf8Error,
+          sync::atomic::{AtomicBool, Ordering}};
+use uuid::Uuid;
+
+// !< constants
+const KMD_PRICE_ENDPOINT: &str = "https://prices.komodo.live:1313/api/v1/tickers";
+
+// !< Type definitions
+pub type StartSimpleMakerBotResult = Result<StartSimpleMakerBotAnswer, MmError<StartSimpleMakerBotError>>;
```

Comment:
We typically use the `Response` or `Res` instead of `Answer`. Could you please use it too for uniformity?